nossas/bonde-bot
# Feature standard-version

# Related issues
- Add support to standard-version #12

# How to test
- Run `yarn install` or `npm install`
- Run `yarn release` or `npm run release`
- A commit and a new tag should be generated automatically by `standard-version` package
